### ISSUE TITLE
android.database.sqlite.SQLiteException: no such table: BOOKS (code 1): , while compiling: SELECT * FROM BOOKS

### ISSUE THREAD

**Issue by TheZoker** [satyan/sugar#629]
I get this issue when I try to start my app.

In `onCreate` I have this:
```SugarContext.init(this);```

**Comment by TheZoker** [comment]
Duplicate of: https://github.com/satyan/sugar/issues/628

**Comment by envoy47** [comment]
I have the same issue and i tried every thing but it doesn't work i'll test other SQLite libraries and choose another one to use.

**Comment by larrytech7** [comment]
Are you calling SugarContext.init() from your Application class or from an Activity?

**Comment by TheZoker** [comment]
Yes I do. After loading the repo instead of the marven project it works.

**Comment by fujinjun** [comment]
@TheZoker you mean,use the git source library instead of gradle dependency？

**Comment by smthAwsm** [comment]
@TheZoker, try to turn off InstantRun. Worked for me.

**Comment by danieltigse** [comment]
I have this issue and I tried everything! The thing is that it occurs in some devices.

**Comment by larrytech7** [comment]
Check that your version of the database is consistent with the changes of
your model objects. Each time to make changes to your model, I think you
need to update your database version in the manifest.

**Comment by jocode** [comment]
Instant-Run seems to prevent Sugar ORM from finding the "table" classes, therefore it cannot create the DB tables if you run the app for the first time

When running your app for the first time Turn off Instant run once to allow for the DB tables to be created You can enable it after the tables have been created.

Ref... Official Repo Sugar
